components/bookshelf/Book.tsx

```diff
@@ -56,6 +69,16 @@ const Book = ({ book, reading = false }: BookProps) => {
         <BookContent>
           <HeaderBar>
             <Title>{book.title}</Title>
+            {(status === StatusEnum.reading || section === "finished") && (
+              <>
+                <Rating
+                  name="simple-controlled"
+                  size="small"
+                  value={currentRating}
+                  onChange={rateBook}
```

Comment:
Don't forget to disable this button when the API request is loading. Else your client may create x number of requests to the backend by clicking the button x number of times. 

Take note of this - one of the things that QAs will test your future features.

Reply:
copy that. will take note. thanks!